py/entry_objects.py

```diff
@@ -16,11 +16,40 @@ class DetailedEntry(Entry):
     A detailed entry includes the title of the event, details regarding logistics,
     and a description of the event.
     Attributes:
-    1. details (list of Link and Text objects) - Each element in the list represents a new detail regarding the event.
+    1. details (list of Details objects) - Each element in the list represents a new detail regarding the event.
     """
     def __init__(self, title, details, body):
         super().__init__(title, body)
         self.details = details
+
+class Description():
+    """
+    Represents a block of text related to an entry's description.
+    Attributes:
+    1. desc (list of Text, Link, and Linebreaks objects) - Each element in the list represents one part of the description.
+    """
+    def __init__(self, desc):
+        self.desc = desc
+
+class Details():
```

Comment:
What's the difference between Description and Details?

Reply:
deleted details